.gitignore

```diff
@@ -15,3 +15,5 @@
 
 # Ignore Byebug command history file.
 .byebug_history
+
+.DS_Store
```

Comment:
You can set up a global ignore file for things like `.DS_Store` that are going to pop up in every project: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore